development/development/git.rst

```diff
@@ -357,7 +380,7 @@ Merging to 3.0.x
 ^^^^^^^^^^^^^^^^
 .. warning::
 
-    **ALL** merges to 3.0.x **must** also be merged to 3.1.x and master!
+    **ALL** merges to 3.0.x **must** also be merged to 3.1.x, 3.2.x, and master!
```

Comment:
Do we even need the Merging to 3.0.x anymore, really?

Reply:
Could be removed, left it in there for now.

Reply:
I say dump it. 3.0 is so 5 years ago :P

Reply:
Ok 🦄